### DISCUSSION TITLE
Cake.EsLint

### DISCUSSION THREAD

**Discussion by pascalberger** [cake-build/cake#2993] [Extension requests]
A plugin for running [ESLint CLI](https://eslint.org/docs/user-guide/command-line-interface) with the option of writing the output to a file

**Comment by nils-a** [comment]
I started working on that under https://github.com/nils-a/Cake.ESlint

**Reply by nils-a** [reply]
First version is shipped
* https://github.com/cake-contrib/Cake.ESLint
* https://www.nuget.org/packages/Cake.ESLint

Currently, there is not much support for options, but I feel the main use case (I.e. start a lint (fully configured using an eslintrc) and writing the results to a file in a specific format) is working.
